be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ElectionHandler.scala

```diff
@@ -34,13 +35,14 @@ object ElectionHandler extends MessageHandler {
   }
 
   def handleSetupElection(rpcMessage: JsonRpcRequestSetupElection): GraphMessage = {
+    //FIXME: add election info to election channel/electionData
```

Comment:
Actually, for validating the votes, we will need to create a new object, Electiondata (similar to LaoData for the use), to check the amount of questions, choices, etc. This will be the subject of another PR.